examples/assembly/addarrays.s

```diff
@@ -0,0 +1,116 @@
+.globl addarrays_serial
```

Comment:
We should follow the lead of the AVX2 example `Sum_array_(AVX2).asm`. How about calling the file `Add_Array_(AArch64).asm`?

Reply:
Also needs a comment with what assembler settings to select:
```
# Assemble with AArch64 Binutils and -march=armv8.2+sve
```
I checked the versions of gas we have and they all support SVE.